src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -40,6 +43,28 @@
     }
 
     /**
+     * Creates a new client that connects to the static server group specified by {@code serverGroup}
+     * using the default {@link RemoteInvokerFactory}.For example,
+     * <p>
+     * <pre>{@code
+     * StaticServerGroup serverGroup = new StaticServerGroup();
+     * serverGroup.addHost("127.0.0.1", 1234, 2).addHost("127.0.0.1", 2345, 4)
+     *     .strategy(StaticServerGroup.SelectStrategyType.ROUND_ROBIN).registry("myGroup");
+     *
+     * Clients.newClient("tbinary+http://myGroup/...", ...,"myGroup",...);
```

Comment:
@trustin Thanks for your code review！The original idea is using a serverGroup name to distinguish the ServerGroup invoke  from the direct URI invoke.Every API's intent is simple.

On the other hand,in order to keep the Clients API's simplicity,maybe we could check serverGroup name automatically and add a little more doc to explain how to use it.what do you think about?